tests/benchmarks/multicore/power_management/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(power_management);
+
+int main(void)
+{
+	int cnt = 0;
+
+	while (1) {
+		LOG_INF("test %d", cnt++);
```

Comment:
logging is irrelevant for PM demo

Reply:
But we may want to print the iterations, do you suggest `printk()` ?

Reply:
Logging is important here, to do basic check if variables/memories are restored correctly after execution ksleep and landing in the idle/sleep mode. Additionaly incrementation of this variable is kind of "heart beat" feature that showing us that CPU is in correct place in case of code execution, escpecially after ksleep and for example does not reset completely.

Reply:
This can be printed together with the printk below, there's no need to add an extra variable to the game (logging subsys). Also, continued printing with a counter can be added to the upstream sample if needed, so that no special code is needed, at least for nRF53. And, for nRF54 we should not need anything else other than `CONFIG_PM=y` and potentially a configurable sleep period over Kconfig so multiple idle states can be easily tested.